pkg/operator/hive/hive_controller.go

```diff
@@ -565,7 +565,7 @@ func (r *ReconcileHiveConfig) Reconcile(ctx context.Context, request reconcile.R
 		return reconcile.Result{}, err
 	}
 
-	err = r.deployHive(hLog, h, instance, namespacesToClean, confighash, managedDomainsConfigHash, fpConfigHash, mcConfigHash)
+	err = r.deployHive(hLog, h, instance, namespacesToClean, confighash, managedDomainsConfigHash, fpConfigHash, mcConfigHash, scConfigHash)
```

Comment:
ntr: trigger redeploy of controllers if supported contracts config changes (e.g. a CRD gets updated to add the new annotation).